### ISSUE TITLE
Add preference to disable syntax highlighting

### ISSUE THREAD

**Issue by patrickjuchli** [sublimehq/sublime_merge#52]
**Problem description**

In the diff view, I only care about two colors/highlights: What was before and what came after. It should be as easy for eyes to keep these two apart. Right now, when looking at Javascript source code for example (see screenshot below), I see way too many different text styles in close proximity which I find confusing and unnecessary.

**Preferred solution**

Disable syntax highlighting aka add preference to disable syntax highlighting in diff view.

**Alternatives**

Use a more subtle syntax highlighting.

**Screenshots**

<img width="599" alt="syntaxdiff" src="https://user-images.githubusercontent.com/397839/46422142-fb68a980-c733-11e8-9966-35f0a247575d.png">

**Comment by frou** [comment]
I agree very much with this.

Syntax highlighting is good. Diff highlighting is good. Both at the same is visual overload.

**Comment by simonbates** [comment]
I would also very much like a preference to turn syntax highlighting on and off. I bought a license for SM as soon as it came out and used it regularly. But I've recently stopped using it as I'm finding the UI too busy. I think it's a combination of more having been added to the UI over time and my personal sensitivity changing over time. In particular, I'm finding the syntax highlighting visually too noisy and it makes it hard to parse the diffs.

**Comment by frou** [comment]
With a bit of work, this is effectively possible to do as a user, by:

* Creating a new [Color Scheme](https://www.sublimetext.com/docs/color_schemes.html) that simply does not target most [Scopes](https://www.sublimetext.com/docs/scope_naming.html).
  * ...or, making a copy of an existing color scheme file and gutting most of it.
* [Telling Sublime Merge to use the new color scheme in the diff pane](https://github.com/sublimehq/sublime_merge/issues/198#issuecomment-471042729).

Here's the end result in my setup, using the same JavaScript code example as the OP:

<img width="513" alt="Screenshot 2022-05-11 at 21 08 18" src="https://user-images.githubusercontent.com/172663/167938247-9ada0e43-dd0c-4f59-a385-0b0e95dcb85b.png">
